api/models/Site.js

```diff
@@ -48,6 +48,12 @@ module.exports = {
       obj.siteRoot = `http://${s3Config.bucket}.s3-website-${s3Config.region}.amazonaws.com`
       obj.viewLink = obj.domain || [obj.siteRoot, 'site', obj.owner, obj.repository].join('/')
 
+      for (key in obj) {
+        if (obj[key] === null) {
+          obj[key] = undefined
+        }
+      }
```

Comment:
Prior to this commit, the in-memory store did not enforce any kind of schema, so values that were not set were undefined. With the Postgres adapter, those become null. When Sails serializes a JSON response, it filters undefined, but not null.

This caused problems with the JSON schemas which expected values to be undefined or of a particular type. It also caused some weird issues in the frontend where keys for values that weren't present where expected to be missing.